### DISCUSSION TITLE
use Template option with out an initial prompt

### DISCUSSION THREAD

**Discussion by ElCid87** [chhoumann/quickadd#503] [Q&A] [answered]
I have a template that uses Templater to prompt for a variable that then get's used in the file title. When I have Quick Add use the template I get prompted twice once when I launch QA and then again when the template runs. 

Is there any way to not have QA prompt?

I did a search for "prompt" and none of the existing discussions were related to this question.

**Comment by chhoumann** [comment, marked as answer]
Hi @ElCid87 

The Templater approach generally works like this:
- Create new note, put template directly into the note
- Render the template syntax, prompting for input, etc.
- In your case: prompt for title, then rename or move the file 

QuickAdd does not support creating notes with no title, as it expects you to facilitate your workflow through it, and not Templater.
QuickAdd will try to 'know everything' _before_ creating the note.

The only workaround I can imagine would be to make the file have a seemingly random file name to begin with, and then let Templater take over once it's ready - e.g. by telling QuickAdd to set the file name as `{{DATE:yyyymmdHHMMss}}` or something. That would become `2023562060520` at the time of writing.

**Reply by ElCid87** [reply]
Ultimately, I renamed the QuickAdd to match the prompt that I had in the template and then just have the template assign the file name to the template variable before renaming and moving the file.

It was just a little frustrating to have to rework some of my templates to account for the fact that I can't just have QA launch a template and have it work the same as ALT-N and select the template.

But I do like the additional capabilities of QA. So it looks like I'll put in the extra effort and look forward to getting more than what I can get with Templater alone.
